infra/launch.py

```diff
@@ -38,44 +40,65 @@ def setup_vm_docker(tpu_name, zone, docker_base_image):
 
 
 def list_tpus(zone):
-    tpus = subprocess.check_output(
-        [
-            "gcloud",
-            "alpha",
-            "compute",
-            "tpus",
-            "tpu-vm",
-            "list",
-            "--zone=" + zone,
-        ]
+    return json.loads(
```

Comment:
Does this also show non-queued-resources TPUs? (Or should we always use queued-resources?)

Reply:
after the migration everything will be in queued resources so we can probably just clean up all the non queued resources tpus and move on.